fontscan/readme.md

```diff
@@ -0,0 +1,76 @@
+# Description and purpose of the package
+
+This package provides a way to locate and load a `font.Font`, which is the
+fundamental object needed by `go-text` for shaping and text rendering.
+
+## Use case
+
+This package may be used by UI toolkits and markup language renderers
+
+### UI toolkits
+
+To support apps displaying a large number of scripts, a GUI toolkit want to leverage
+the fonts provided by the OS (system fonts). This ensure correct rendering, and alleviate
+the work required by the developper.
+
+### Markup language renderers
+
+The second use case deals with the rendering of markup documents (think HTML or SVG), where the author provide _hints_ about the fonts that should be used. In this case, the two main requirements are
+
+- use the whole system fonts to find the best match with respect to the author intention
+- handle a large rune coverage : many scripts may be present in the same document, so that we can't specify at build time the set of fonts we will use
```

Comment:
These use-cases aren't all that different anymore, so we can probably merge these sections.